libjwt/jwt.c

```diff
@@ -250,7 +262,7 @@ static long get_js_int(json_t *js, const char *key)
 	return val;
 }
 
-static void *jwt_b64_decode(const char *src, int *ret_len)
+void *jwt_b64_decode(const char *src, int *ret_len)
```

Comment:
It was tagged as a conflict. The only difference I could spot was the static key word.